packages/dds/matrix/src/test/memory/sharedMatrix.spec.ts

```diff
@@ -52,229 +136,448 @@ describe("SharedMatrix memory usage", () => {
 
 			// Insert-related tests that are not limited by matrixSize
 			for (const count of operationCounts) {
-				// Test the memory usage of the SharedMatrix for inserting a column in the middle for a given number of times.
-				benchmarkMemory(
-					new (class implements IMemoryTestObject {
-						readonly title = `Insert a column in the middle ${count} times`;
-						private localMatrix: SharedMatrix | undefined;
-
-						async run(): Promise<void> {
-							if (this.localMatrix === undefined) {
-								throw new Error("localMatrix is not initialized");
+				describe(`Column Insertion`, () => {
+					benchmarkMemory(
```

Comment:
We seem to have lost the individual test descriptions. I think we should restore them. The format of `benchmarkMemory` doesn't make the title obvious at-a-glance, so having descriptions above each one seems important.

Reply:
Restore the comments